test/test_lambda.py

```diff
@@ -0,0 +1,41 @@
+from app import lambda_handler
+
+
+def test_root_returns_html():
```

Comment:
Is it possible to test the status as well?

Reply:
Yeah, you can. I've updated the tests to include it.